lib/app/modules/groups/presentation/pages/group_management_page.dart

```diff
@@ -111,80 +114,150 @@ class GroupManagementPage extends StatelessWidget {
                         ),
                       ),
                       const SizedBox(height: 48),
-                      ZiggleRowButton(
-                        showChevron: true,
-                        title: Text(context.t.group.manage.name.header),
-                        onPressed: () => GroupManagementNameRoute(
-                          uuid: group.uuid,
-                          name: group.name,
-                        ).push(context),
+                      BlocBuilder<GroupRoleBloc, GroupRoleState>(
+                        builder: (context, state) {
+                          final disabled = state.maybeWhen(
+                            loaded: (role) =>
+                                role == GroupMemberRole.admin ? false : true,
+                            orElse: () => true,
+                          );
```

Comment:
헷갈려서 이런방식을 사용한 거면 아래처럼 enabled를 구하고 disabled에서는 !enabled로 적으면 돼요
```suggestion
                          final enabled = state.maybeWhen(
                            loaded: (role) => role == GroupMemberRole.admin,
                            orElse: () => true,
                          );
```

Reply:
GroupMemberRole에 메소드를 만드는 것도 좋아보여요

Reply:
그리고 나중에는 rbac로 구현해야하는거 염두해두세요 (permission 가지고 컨트롤)

Reply:
넹